bindings/nodejs/types/account.ts

```diff
@@ -10,6 +10,7 @@ import type { Transaction } from './transaction';
 export type AccountId = number | string;
 
 export interface AccountBalance {
+    baseCoin: BaseCoinBalance;
     total: string;
     available: string;
     requiredStorageDeposit: string;
```

Comment:
Can't comment there, but `nativeTokens` also changed to have the total and available fields now